utils/gitlab_api.py

```diff
@@ -188,6 +188,60 @@ def create_app_interface_reporter_mr(self, reports, email_schema,
         return self.create_mr(branch_name, target_branch,
                               commit_message, labels=labels)
 
+    def create_app_interface_notificator_mr(self, notifications,
+                                             notification_body,
+                                             notification_path="/app-interface",
+                                             email_schema="/app-interface/app-interface-email-1.yml"):
+        # labels = ['automerge']
+        labels = ['do-not-merge/hold']
+        prefix = 'app-interface-notificator'
+        target_branch = 'master'
+
+        now = datetime.now()
+        ts = now.strftime("%Y%m%d%H%M%S")
+        isodate = now.isoformat()
+        short_date = now.strftime('%Y-%m-%d')
+
+        branch_name = 'app-interface-notificator-{}'.format(ts)
+        commit_message = '[{}] notification for {}'.format(
+            prefix, isodate
+        )
+
+        self.create_branch(branch_name, target_branch)
+
+        actions = [
+            {
+                'action': 'create',
+                'file_path': notification['file_path'],
+                'content': notification['content'],
+            }
+            for notification in notifications
+        ]
+
+        self.create_commit(branch_name, commit_message, actions)
+
+        msg = 'add email notification'
+        email_path = f"data{notification_path}/emails/{branch_name}.yml"
+        email = {
```

Comment:
the comments in this dict seem a bit off. can you review and remove the comments or parts that are not needed?